openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -920,6 +926,23 @@ Below is a non-normative example of a `proof` parameter:
 
 ```
 
+#### `attestation` Proof Type {#attestation-proof-type}
+
+A key attestation in JWT format as defined in (#keyattestation-jwt).
+
+When a key attestation is used as a proof type, it MUST contain the `c_nonce` value provided by the Credential in its `nonce` parameter.
+
+Below is a non-normative example of a `proof` parameter (with line breaks within values for display purposes only):
+
+```json
+{
+  "proof_type": "attestation",
+  "attestation": "<key attestation in JWT format>"
+}
+```
+
+The Credential Issuer SHOULD return a Credential for each of the keys provided in the `attested_keys` claim of the `attestation`.
```

Comment:
Why only SHOULD?

Reply:
There might be cases where it seems unreasonable to do so - for example if the wallet is providing 200 key attestations.

To be fair this doesn't really seem to be a problem with the attestation itself, but to the best of my knowledge, we do not have any normative language in VCI right now that the Issuer MUST answer a request with all requested credentials. I guess it would make sense to create an issue to discuss this and then fix the language in credential response and proof types?